lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -191,14 +191,23 @@ defmodule LivebookWeb.SessionLive.CellComponent do
           <div class="input-label">
             <%= @cell_view.name %>
           </div>
-          <input type="<%= if(@cell_view.input_type == :password, do: "password", else: "text") %>"
-            data-element="input"
-            class="input <%= if(@cell_view.error, do: "input--error") %>"
-            name="value"
-            value="<%= @cell_view.value %>"
-            spellcheck="false"
-            autocomplete="off"
-            tabindex="-1" />
+          <%= if (@cell_view.input_type == :textarea) do %>
+            <textarea
```

Comment:
Do we want the `autocomplete="off"` attribute for the textarea too?

Alternatively, looks like those values can be set at the form level, which may be nice if that behavior should apply to all elements.

Reply:
The autocomplete for input and select works differently.

Using the option in the form is maybe a good idea, but I think it's better to discuss this in another PR